pkg/cmd/admin/router/router.go

```diff
@@ -591,6 +599,10 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		"STATS_USERNAME":                      cfg.StatsUsername,
 		"STATS_PASSWORD":                      cfg.StatsPassword,
 	}
+	if len(cfg.ForceSubdomain) > 0 {
+		env["ROUTER_SUBDOMAIN"] = cfg.ForceSubdomain
+		env["ROUTER_OVERRIDE_HOSTNAME"] = "true"
```

Comment:
Flags fail on old images, env gracefully degrades.  That and we're
already using env consistently.  Felt worse to change it all.